src/js/main.js

```diff
@@ -0,0 +1,81 @@
+export const DiscourseRADParser = () => {
+  const radTabsElements = document.querySelectorAll('.rad-tabs-element');
+
+  radTabsElements.forEach((radTabsElement) => {
+    setDefaults(radTabsElement);
+    setupDropdowns(radTabsElement);
+  });
+
+  toggleElements();
+}
+
+const setDefaults = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
```

Comment:
Can you target a class here? There may be a point in the future where we switch to another element and that would cause unexpected failures here.

Reply:
I did have a common class for all my select elements, so it was not a big deal to replace it. But I am not sure it makes it any more fail safe.